### PR TITLE
updated ansible versions in meta and updated ansible-test workflow

### PR DESCRIPTION
**Pull request by djdanielsson**
modified:   roles/analysis/meta/main.yml
modified:   roles/common/meta/main.yml
modified:   roles/parse_leapp_report/meta/main.yml
modified:   roles/remediate/meta/main.yml
modified:   roles/upgrade/meta/main.yml
modified:   .github/workflows/ansible-test.yml
modified:   roles/analysis/files/check-inodes.sh

### CODE REVIEW

File: meta/runtime.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
-requires_ansible: ">=2.15.0" # Use '>= 2.9.10' instead, if needed
+requires_ansible: ">=2.15.0" # Use '>= 2.11.0' instead, if upgrading RHLE 6
```
Comment (swapdisk) [inline meta/runtime.yml:2]: s/RHLE/RHEL/

File: roles/analysis/files/check-inodes.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -eu
```
Comment (swapdisk) [inline roles/analysis/files/check-inodes.sh:1]: Just curious why this change?

**Review by swapdisk — CHANGES_REQUESTED** [review]
Fix typo. Otherwise, LGTM.

File: roles/analysis/files/check-inodes.sh (hunk 2 above)
Comment (djdanielsson) [reply roles/analysis/files/check-inodes.sh:1]: Idk it's what it suggested to fix it

File: meta/runtime.yml (hunk 1 above)
Comment (djdanielsson) [reply meta/runtime.yml:2]: lol oops

**Review by swapdisk — APPROVED** [review]
LGTM.
